packages/flutter/lib/src/foundation/print.dart

```diff
@@ -31,6 +31,14 @@ typedef DebugPrintCallback = void Function(String? message, { int? wrapWidth });
 /// Prints a message to the console, which you can access using the "flutter"
 /// tool's "logs" command ("flutter logs").
 ///
+/// `debugPrint` logs to console even in [release mode](https://docs.flutter.dev/testing/build-modes#release).
```

Comment:
nit: "Avoid starting a sentence with a lowercase letter.", https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar